src/app/app.component.scss

```diff
@@ -0,0 +1,7 @@
+
+.box{
+  margin: auto;
+  width: 70vw;
```

Comment:
please don't use vw and wh here. If you need padding/margin add it with margin/padding + px/rem